unity/Assets/Maroon/scenes/experiments/3DMotionSimulation/Scripts/ParameterLoader.cs

```diff
@@ -32,47 +28,42 @@ public class Parameters
         public float Vy;
         public float Vz;
 
+        public string m;
+        public string fx;
+        public string fy;
+        public string fz;
+
+        public Dictionary<string, string> exprs = new Dictionary<string, string>();

```

Comment:
To be honest, I am not a big fan of abbreviating variable names, E.g. writing` expressions` instead of `exprs` doesn't take that much longer with auto-completion of most IDEs, and code would become more readable.